src/renderer/css/component/tooltip.css

```diff
@@ -0,0 +1,79 @@
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltip .tooltip-text {
+  visibility: hidden;
```

Comment:
do we need this? Maybe just show / hide using `display` and remove `visibility` and `opacity`?

Reply:
Probably not. Changed it to your suggestion.